ethcore/machine/src/test_helpers.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity Ethereum.
+
+// Parity Ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
+
+// Creating test Machines
+
+use common_types::engines::params::CommonParams;
+use ethjson;
+use crate::Machine;
+
+
+pub fn load_machine(reader: &[u8]) -> Machine {
```

Comment:
this will cause conflicts with #10948 . not an issue, just observation :)

Reply:
Yes, it'll be terrible!